meeg_tools/utils/log.py

```diff
@@ -49,7 +49,6 @@ def update_log(log_file_path: str, epochs: Epochs, notes: str) -> pd.DataFrame:
             "t_min": [epochs.tmin],
             "t_max": [epochs.tmax],
             "n_interpolated": [np.NaN],
-            "excluded_components": ica.exclude,
```

Comment:
this change is rolled back because there is no ica object in the update_log function, this would result in error